go/interpreter/lfvm/gas.go

```diff
@@ -297,7 +327,7 @@ func gasSStore(c *context) (tosca.Gas, error) {
 //     2.2.2.2. Otherwise, add SSTORE_RESET_GAS - SLOAD_GAS gas to refund counter.
 func gasSStoreEIP2200(c *context) (tosca.Gas, error) {
 	// If we fail the minimum gas availability invariant, fail (0)
-	if c.gas <= tosca.Gas(params.SstoreSentryGasEIP2200) {
+	if c.gas <= tosca.Gas(SstoreSentryGasEIP2200) {
```

Comment:
The `tosca.Gas(...)` cast would not really be needed here since constants are of the exotic "untyped int" type.

However, we could enforce that all those constants are Gas values (and thereby avoid accidental miss-use) by defining them all as gas values like this:
```
SstoreResetGasEIP2200      tosca.Gas = 5000
```
or
```
SstoreResetGasEIP2200      = tosca.Gas(5000)
```
Wdyt?